scripts/tank_cmd_login.sh

```diff
@@ -64,6 +64,9 @@ then
     interpreter=$( cygpath -u $interpreter )
 fi
 
+# Expand environment variables
+interpreter=`eval echo $interpreter`
```

Comment:
Can we do the same thing for Windows and it's `.bat` file?

Reply:
I can't test on Windows, so I can't do this properly.

Reply:
Alright, we'll make sure to fix this on our end!